include/deal.II/lac/ginkgo_solver.h

```diff
@@ -219,11 +225,294 @@ namespace GinkgoWrappers
      * parameters and setup the CG solver from the CG factory which solves the
      * linear system.
      *
-     * @p executor The execution paradigm for the CG solver.
+     * @p exec_type The execution paradigm for the CG solver.
+     */
+    SolverCG(SolverControl &       solver_control,
+             std::string           exec_type,
```

Comment:
make it a `const` reference. this is our usual style for non-scalar input arguments